src/main/java/edu/cornell/kfs/sys/businessobject/lookup/ApiFrameworkLookupableHelperServiceImpl.java

```diff
@@ -0,0 +1,27 @@
+package edu.cornell.kfs.sys.businessobject.lookup;
+
+import java.util.List;
+
+import org.apache.commons.lang3.StringUtils;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.kuali.kfs.kns.lookup.HtmlData;
+import org.kuali.kfs.kns.lookup.KualiLookupableHelperServiceImpl;
+import org.kuali.kfs.krad.bo.BusinessObject;
+import org.kuali.kfs.krad.util.KRADConstants;
+
+public class ApiFrameworkLookupableHelperServiceImpl extends KualiLookupableHelperServiceImpl{
```

Comment:
Missing space prior to the curly brace.

Also, this customization might not prevent users from manually/forcibly browsing to the "copy" URL. However, given that this document can only be created by a small number of users anyway, removing the "copy" link from the search results should be enough for this case.

Reply:
ok